bundles/org.openhab.binding.heliosventilation/src/main/resources/datapoints.properties

```diff
@@ -0,0 +1,48 @@
+#
+# datapoints.properties - This file defines the datapoints of the Helios ventilation system
+#
+# Format: <name> = <address(:bitspec)>,writable,type
+#
+#         bitspec is
+#         - a single digit in range 0-7 or
+#         - start:end (where start is the number of the LSB and end the number of the MSB of the field)
+#
+#         type is one of
+#         - TEMPERATURE
+#         - FANSPEED
+#         - PERCENT
+#         - BYTE_PERCENT
+#         - SWITCH
+#         - NUMBER
+#         - HYSTERESIS
+#
+# on change of this file, ensure that the thing-types.xml is consistent
+
+fanspeed = 0x29,true,FANSPEED
```

Comment:
Is there a reason that you didn't store this data as an enum in the source code?

Reply:
initially it was hardcoded, but during the reviews of the first version of this PR (#5831) it was recommended to move it into a properties file.